rust/openvasd/src/config.rs

```diff
@@ -597,8 +598,8 @@ impl Config {
         if let Some(path) = cmds.get_one::<PathBuf>("tls-client-certs") {
             config.tls.client_certs = Some(path.clone());
         }
-        if let Some(enable) = cmds.get_one::<bool>("enable-get-scans") {
-            config.endpoints.enable_get_scans = *enable;
```

Comment:
Since we load from the file first, wouldn't it be possible to check if `config.endpoints.enable_get_scans` is false and only when it is false call `cmds.get_one::<bool>("enable-get-scans")`? That way we could still stick to `--enable-get-scans` without having to add an additional value to that.

Reply:
You don't need to add an additional value. Just running `openvasd --enable-get-scans` will set this automatically to `true`, because of the `.default_missing_value("true")`